<ui version="4.0" >
 <class>TableDialogUI</class>
 <widget class="QDialog" name="TableDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Class Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mName" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Table Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="textLabel5" >
       <property name="text" >
        <string>Primary Key Hash Size:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="mClass" />
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="mCacheSizeSpin" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mUseCodeGen" >
     <property name="text" >
      <string>Generate C++ Class</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mCreatePKeyCheck" >
     <property name="text" >
      <string>Create Primary Key</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mPreload" >
     <property name="text" >
      <string>Preload Data:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mExpireKeyCacheCheck" >
     <property name="text" >
      <string>Expire Key Cache</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="mProjectPreload" >
       <property name="text" >
        <string>Project Preload:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mFieldCombo" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mEditDocsButton" >
     <property name="text" >
      <string>Edit Documentation...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>41</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>mOK</sender>
   <signal>clicked()</signal>
   <receiver>TableDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel" >
     <x>122</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>TableDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>289</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
